python/dgl/contrib/dis_kvstore.py

```diff
@@ -296,6 +336,40 @@ def get_message_count(self):
         return self._msg_count
 
 
+    def get_data_name_list(self):
```

Comment:
do we need to define this on the kvstore server?

Reply:
As written in our quip doc: All the information should be available in the kvstore server. So I remain this API on server.

Reply:
I see. By "All the information should be available in the kvstore server", what I mean is that the information is available on the server, so the client can get the information from the server directly. I didn't mean we should provide this API on the server as well.